### PR TITLE
Multiple basestations

### PR DESCRIPTION
**Pull request by Florisheinen1**
Previously, only one Basestation could be connected to a RobotHub. This PR allows for an infinite amount of Basestations, of which 2 will automatically be selected.

This unfortunately does not change the frequencies of the Basestations, so make sure you hard-code one of the Basestations to use the blue channel and one to use the yellow channel.
This changing of frequencies will be later implemented.

This PR also makes sure RobotHub uses the World included in the AICommand, instead of using the one published by Observer, as apparently, the AI world does not result in segfaults.
This functionality will be removed in the future however, as it does not make sense to parse the world inside RobotHub

### CODE REVIEW

File: src/RobotHub.cpp [hunk 1]
```diff
@@ -7,6 +7,13 @@
 
 namespace rtt::robothub {
 
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_BLUE_CONTROL = 30011;
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_YELLOW_CONTROL = 30012;
+constexpr int DEFAULT_GRSIM_FEEDBACK_PORT_CONFIGURATION = 30013;
+
+constexpr float DEFAULT_CHIPPER_ANGLE = 45.0f;
+constexpr float MAX_DRIBBLER_SPEED = 1021.0f;
+
```
Comment (emielsteerneman) [inline src/RobotHub.cpp:17]: ^ Are these two lines for GRSIM? If so, please prepend with GRSIM_ or something.

**Review by emielsteerneman — APPROVED** [review]
It's quite a lot of changes, but it looks good. Impressive stuff figuring out the USB library. It took me weeks to put down something functional, that library is horrible.

This code does however need much more documentation, and maybe a flowchart.

File: src/RobotHub.cpp [hunk 2]
```diff
@@ -181,7 +193,7 @@ void RobotHub::printStatistics() {
 
 void RobotHub::handleRobotFeedbackFromSimulator(const simulation::RobotControlFeedback &feedback) {
     proto::RobotData feedbackToBePublished;
-    feedbackToBePublished.set_isyellow(feedback.isTeamYellow);
+    feedbackToBePublished.set_isyellow(feedback.color == utils::TeamColor::YELLOW ? true : false);
```
Comment (alexander-de-ranitz) [inline src/RobotHub.cpp]:
```suggestion
    feedbackToBePublished.set_isyellow(feedback.color == utils::TeamColor::YELLOW);
```

Comment (Florisheinen1) [reply src/RobotHub.cpp]: Oops, Ill fix that right away

File: src/RobotHub.cpp (hunk 1 above)
Comment (Florisheinen1) [reply src/RobotHub.cpp:17]: These variables describe properties of our physical robots. These are indeed used in commands for simulators. These values were used in previous year, so I do not know the correctness of them.

I changed the names a bit and added comments. I think that should be sufficient for now at least

**Pull request merged**